### ISSUE TITLE
print expression fails to parse

### ISSUE THREAD

**Issue by heycam** [Pernosco/pernosco#9]
I'm having trouble with print expressions in the executions view.  I am debugging Firefox and have executions of `SVGGeometryFrame::PaintSVG` listed, and I want to print out the value of the CSS fill property for each execution.  I tried using this (long) expression:

```
this->mComputedStyle.mRawPtr->mSource.SVG.mPtr->gecko.mInner.mFill.kind.color._0.numeric._0.red
```

which I think adheres to the restrictions in the help page, but it's having trouble parsing the expression.  The underline starts at `.mRawPtr->mSource...`.

**Comment by khuey** [comment]
Can you provide us a Pernosco link as well?

**Comment by heycam** [comment]
https://pernos.co/debug/X8pDMDNi-GN8Txh6AAurQg/index.html#f{m[2Y4,G6e9_,t[AQ,H2E_,f{e[2Y4,GunE_,s{af98NexAA,bAXU,oBlQvGA,uBkbGFw___

**Comment by khuey** [comment]
Fixed by d61eeecf.
